picker/blueprinter.lua

```diff
@@ -83,9 +83,9 @@ Event.register(defines.events.on_put_item, last_item)
 -------------------------------------------------------------------------------
 --[[Make Simple Blueprint]]--
 -------------------------------------------------------------------------------
-local function make_simple_blueprint(event)
```

Comment:
For the moment I will have to turn this down,  Picker is an exercise in what can be done without a tick handler. Also I am hoping for player.pipette() to be a thing soon.